content/developer/reference/views.rst

```diff
@@ -1505,6 +1505,11 @@ Possible children of the view element are:
   ``name`` (required)
     the name of the field to fetch
 
+  ``allow_group_range_value`` (optional)
+    whether a ``date`` or ``datetime`` field allows a value computed from a
+    group range (which consists of the first and last dates of the group).
+    Enables the 'quick create' and 'drag and drop' features. Default: false.
```

Comment:
Enables the 'quick create' and 'drag and drop' features when the kanban view is grouped by that field.